command-palette/README.md

```diff
@@ -0,0 +1,72 @@
+# Command Palette: Extending the main app
+
+One major concept of the phosphorjs library on which JupyterLab is built is
```

Comment:
Should we start using [Lumino](https://github.com/jupyterlab/lumino) directly, or wait a little bit longer?

Reply:
I think you should stick to JLab 1.2 for now (and phosphorjs). And make the switch for JLab 2.0 release. What do you think? @echarles

Reply:
Sounds good, at least wait for the switch to happen in JLab.